packages/cdk8s/test/chart.test.ts

```diff
@@ -93,6 +93,57 @@ test('synthesizeManifest() can be used to synthesize a specific chart', () => {
   expect(manifest).toMatchSnapshot();
 });
 
+describe('toJson', () => {
+
+  test('returns an ordered list', () => {
+
+    const app = Testing.app();
+    const chart1 = new Chart(app, 'chart1');
+
+    const obj1 = new ApiObject(chart1, 'obj1', { apiVersion: 'v1', kind: 'Kind1' });
+    const obj2 = new ApiObject(chart1, 'obj2', { apiVersion: 'v1', kind: 'Kind2' });
+    const obj3 = new ApiObject(chart1, 'obj3', { apiVersion: 'v1', kind: 'Kind3' });
+
+    Node.of(obj1).addDependency(obj2);
+    Node.of(obj2).addDependency(obj3);
+
+    expect(chart1.toJson()).toEqual([obj3.toJson(), obj2.toJson(), obj1.toJson()]);
```

Comment:
Style: put each array element in a new line (especially here where it’s all about the order)

Reply:
👍